### PR TITLE
Fix Distro on Windows

### PR DESCRIPTION
**Pull request by philippfromme**
Assets weren't getting copied to `dist` and no test verified them being copied. `v0.13.0` was shipped without some of the assets.

### CODE REVIEW

**Comment by nikku** [comment]
Can we mark this as fixing https://github.com/camunda/camunda-bpmn-js/issues/105?

**Review by nikku — COMMENTED** [review]
Very very nice!

Added one minor comment for your consideration.

File: tasks/test-distro.js [hunk 1]
```diff
@@ -45,12 +66,26 @@ function runTest(variant, env) {
   }
 }
 
+function testAssets(assets) {
+  assets.forEach(asset => {
+    try {
+      fs.accessSync(path.resolve('./dist/assets/', asset));
+    } catch (err) {
+      console.log('asset', asset, 'not found');
```
Comment (nikku) [inline tasks/test-distro.js:74]:
```suggestion
      console.error('asset', asset, 'not found');
```

[the remark this answers is not in the document]
Comment (philippfromme) [reply tasks/test-distro.js]: Done!

Comment (philippfromme) [reply tasks/test-distro.js]: Done!

**Comment by philippfromme** [comment]
@nikku Will `console.error` in case an asset wasn't found. 👍🏻

**Review by nikku — APPROVED** [review]
Marvelous!

**Pull request merged**